### PR TITLE
Added manual bridge and things configuration

### PR DESCRIPTION
**Pull request by udo1toni**
finally...

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: 2
 
 requirements:
-  signed_off_by:
+  signed_off_by: Udo Hartmann <udo1toni@gmail.com>
```
Comment (martinvw) [inline .pullapprove.yml]: Please leave this file alone

Comment (martinvw) [reply .pullapprove.yml]: https://pullapprove.com/openhab/openhab2-addons/pull-request/2710/

Comment (udo1toni) [reply .pullapprove.yml]: I suspect I did not understand how to sign off the correct way.
Please bring light to the darkness...

**Comment by udo1toni** [comment]
Next try...

**Pull request closed without merging**